etl/steps/data/garden/oecd/2024-08-21/official_development_assistance.meta.yml

```diff
@@ -0,0 +1,933 @@
+  key_officially_supported_export_credits_1: Governments provide officially supported export credits through Export Credit Agencies (ECAs) in support of national exporters competing for overseas sales. ECAs can be government institutions or private companies that operate on behalf of governments.
+  key_officially_supported_export_credits_2: Such support can take the form either of “official financing support”, such as direct credits to foreign buyers, refinancing or interest-rate support, or of “pure cover support”, such as export credits insurance or guarantee to cover credits provided by private financial institutions.
+  key_private_flows_market_terms: |-
+    Private flows are financed out of private sector resources (changes in holdings of private, long-term assets held by residents of the reporting country) and private grants (grants by non-government organizations, net of subsidies received from the official sector). Private capital flows can be divided into: foreign direct investment; portfolio equity (the buying and selling of stocks and shares); remittances sent home by migrants; and private sector borrowing.
+  key_net_private_grants: The private sector comprises private corporations, households and non-profit institutions serving households. Development funding from the private sector is becoming more significant. This includes private foundations, which play an increasing role in funding development and in finding innovative ways to promote it; non-government organisations; and the for-profit private sector.
+
+  key_flows: Total official flows incorporate the sum of concessional and non-concessional flows to developing countries, including export credits, which have a primarily commercial motive. Private flows are defined as flows at market terms and financed out of private sector resources and private grants.
+
+  key_oda_bilateral: Bilateral ODA also includes cases where a donor contracts a multilateral agency to deliver a program or project on its behalf. This is often referred to as “Bi/Multi” flows. Donor funds channeled through NGOs are also typically counted as bilateral flows.
+
+  key_food_aid: |-
+    In terms of its use, food aid can be subdivided into three categories: program food aid (budgetary support), project food aid (targeted poverty alleviation and disaster prevention) and relief food aid (targeted to victims of disasters). It can also be classified by modes of suppply in direct transfers, food aid purchases or local purchases.
+
+  key_sector: |-
+    <% if sector == "Humanitarian aid" %>
+    Humanitarian aid broadly includes aid given for emergencies, such as natural disasters and wars, reconstruction in their aftermath, but also prevention and preparation for future emergencies.
+    <% elif sector == "Non-humanitarian aid" %>
+    Non-humanitarian aid broadly is aid for longer-term development, such as social and economic infrastructure.
+    <% else %>
+    ""
```

Comment:
@Marigold Hi! Would there be any way to leave some texts on a description_key list empty and not show them in Grapher?

Reply:
This is currently the outcome for the sectors in "else":
<img width="558" alt="image" src="https://github.com/user-attachments/assets/8db42fa9-463c-42f7-a8fb-0732c5e97b2c">

Reply:
Oh, it seems I can do it without specifying an else!

Reply:
<img width="567" alt="image" src="https://github.com/user-attachments/assets/33dc5b1b-978d-49ab-915c-4fc0a94294f7">